### PR TITLE
Test snapshot export to s3

### PR DESCRIPTION
**Pull request by siegfriedweber**
# Description

Test the export of a snapshot to S3 using the script `export-snapshot-to-s3` contained in the HBase image.

Tests stackabletech/docker-images#621

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [x] Changes are OpenShift compatible
- [x] Integration tests passed (for non trivial changes)
- [ ] Changes need to be "offline" compatible
```

```[tasklist]
# Reviewer
- [x] Code contains useful comments
- [x] Code contains useful logging statements
- [x] (Integration-)Test cases added
- [x] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [x] Changelog updated
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

File: tests/templates/kuttl/snapshot-export/31-run-tests.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,41 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: test-snapshot-export
+spec:
+  template:
+    spec:
+      containers:
+        - name: hbase
+          image: docker.stackable.tech/sandbox/hbase:{{ test_scenario['values']['hbase'] }}-stackable0.0.0-pr621
```
Comment (siegfriedweber) [inline tests/templates/kuttl/snapshot-export/31-run-tests.yaml.j2:10]: Change when stackabletech/docker-images#621 is merged:

```suggestion
          image: docker.stackable.tech/sandbox/hbase:{{ test_scenario['values']['hbase'] }}-stackable0.0.0-dev
```

**Comment by siegfriedweber** [comment]
https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hbase-operator-it-custom/141/

**Review by adwk67 — APPROVED** [review]
LGTM - tested on Openshift (OKD 4.15) and AKS.

File: tests/templates/kuttl/snapshot-export/31-run-tests.yaml.j2 (hunk 1 above)
Comment (siegfriedweber) [reply tests/templates/kuttl/snapshot-export/31-run-tests.yaml.j2]: Fixed in 7dd1592eb993453b1ed6f3cf8fed5ba77c0ba531

**Comment by siegfriedweber** [comment]
https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/hbase-operator-it-custom/143/

**Review by adwk67 — APPROVED** [review]

**Pull request merged**